src/lavatory/commands/purge.py

```diff
@@ -36,15 +37,11 @@
 def purge(ctx, dryrun, policies_path, default, repo, repo_type):
     """Deletes artifacts based on retention policies."""
     LOG.debug('Passed args: %s, %s, %s, %s, %s, %s', ctx, dryrun, policies_path, default, repo, repo_type)
-    artifactory = Artifactory(repo_name=None)
-    before_purge_data = artifactory.list(repo_type=repo_type)
-    if repo:
-        selected_repos = repo
-    else:
-        selected_repos = before_purge_data.keys()
+
+    storage_info, selected_repos = get_artifactory_info(repo_name=None, repo_type=repo_type)
```

Comment:
If this only ever passes `repo_name = None`, how does it get repos specified in the CLI?